composer.json

```diff
@@ -13,7 +13,9 @@
         "sparkphp/command": "self.version"
     },
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "equip/assist": "^0.1.2",
+        "equip/data": "^2.3"
```

Comment:
That doesn't answer my question. Do you have specific use cases in mind wherein we would use immutable traits outside of repositories and entities? If so, do you anticipate conflicts happening with sufficient frequency to justify maintaining yet another repository?